onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql

```diff
@@ -1,4 +1,9 @@
+<<<<<<< HEAD
+--Update to use corredct method to cross container
+--jonesga 11/5/2021
+=======
```

Comment:
Does this line marked as a sql comment like adding "---" three  hyphen symbol preceding your text?  If not, then this will show as an error as sql statement.